File: UCosmic.Web.Mvc/scripts/viewmodels/representatives/RepModuleSettings.js
var ViewModels;
(function (ViewModels) {
    var Representatives;
    (function (Representatives) {
        var RepModuleSettings = (function () {
            function RepModuleSettings(welcomeMessage, emailMessage) {
                this.welcomeMessage = welcomeMessage;
                this.emailMessage = emailMessage;
            }
            return RepModuleSettings;
        })();
        Representatives.RepModuleSettings = RepModuleSettings;
    })(Representatives = ViewModels.Representatives || (ViewModels.Representatives = {}));
})(ViewModels || (ViewModels = {}));

File: UCosmic.Domain/Domain/Representatives/Entities/RepModuleSettings.cs
using System.Collections.Generic;
using UCosmic.Domain.Establishments;

namespace UCosmic.Domain.Representatives
{
    public class RepModuleSettings : Entity
    {
        protected internal RepModuleSettings()
        {
        }

        public int Id { get; protected set; }
        public string WelcomeMessage { get; protected internal set; }
        public virtual Establishment Owner { get; protected internal set; }
        public virtual ICollection<ApplicationRecipient> ApplicationRecipients { get; protected internal set; }
    }
}
